content/01.main_content.md

```diff
@@ -1,20 +1,19 @@
-Recognition by peers or the general public can greatly affect the trajectory of a scientist’s career.
-  Whether the recognition is the inclusion of their name in print as an expert, an invited lecture, or an award, each form of recognition paves the way to future accolades.
-  Regardless of an individual’s merit, biases can skew which scientists are even considered for recognition.
-  To combat biases, audits can identify if specific groups are being unintentionally neglected and hold the recognizing body accountable.
+Recognition by peers or the general public can greatly affect the trajectory of a scientist’s career since each form of recognition paves the way to future accolades.
+  Regardless of an individual’s merit, biases can skew which scientists are considered for recognition.
+  Audits can combat this by identifying neglected groups and hold the recognizing body accountable.
 
-The use of auditing methods has produced evidence that they can help reduce disparities.
-  In science journalism, Adrienne LaFrance, supported by Nathan Matias, and Ed Yong performed self-audits to quantify gender disparity in their quoted or mentioned sources [@https://www.theatlantic.com/technology/archive/2016/02/gender-diversity-journalism/463023/; @https://www.theatlantic.com/science/archive/2018/02/i-spent-two-years-trying-to-fix-the-gender-imbalance-in-my-stories/552404/].
-  While a secondary audit performed by LaFrance showed no improvement from her initial audit, Yong, through continuous self-auditing, was able to achieve gender parity in his reporting.
+Auditing efforts have shown that they can help reduce disparities.
+  In science journalism, Adrienne LaFrance, supported by Nathan Matias, and Ed Yong performed self-audits to quantify gender disparity in their sources [@https://www.theatlantic.com/technology/archive/2016/02/gender-diversity-journalism/463023/; @https://www.theatlantic.com/science/archive/2018/02/i-spent-two-years-trying-to-fix-the-gender-imbalance-in-my-stories/552404/].
+  While a secondary audit performed by LaFrance showed no improvement, Yong, through continuous self-auditing, was able to achieve gender parity in his reporting.
 
-In award recognition, an audit of the International Society for Computational Biology (ISCB) honorees revealed a significant disparity against people with East Asian name origins and towards US-affiliated scientists.
-  After this study was made available to the public, the following set of honorees had the highest mean predicted probability of an honoree having an East Asian name of any previous year.
-  Additionally, the nominating committee inducted the first China-based fellow into ISCB [@doi:10.1016/j.cels.2021.07.007].
+In award recognition, an audit of the International Society for Computational Biology (ISCB) honorees revealed a significant under-representation of people with East Asian name origins and under-representation of US-affiliated scientists.
```

Comment:
over-representation you mean?

Reply:
oops, fixed: https://github.com/greenelab/disparities_opinion/pull/8/commits/4844c380c9cf2566cdf98c5c6c31833a8492cf7f